GuildedRoseLLC/Models/Item.swift

```diff
@@ -41,3 +53,37 @@ extension Item {
     ]
     static var singleTestItem = Item(name: "Foo")
 }
+
+
+
+class ItemRepository {
```

Comment:
I really like the interface to this class. There's a chance it might need to change if/when we add error handling in, but I think it's great to keep it relatively simple for now.